src/main/java/Database/DatabaseGraphController.java

```diff
@@ -0,0 +1,124 @@
+package Database;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import pathfinding.Floor;
+import pathfinding.HospitalEdge;
+import pathfinding.HospitalNode;
+import pathfinding.MoveAttribute;
+import pathfinding.NodeInitializer;
+
+public class DatabaseGraphController {
+  DatabaseController DBC;
+  private static List<HospitalNode> nodeList = new ArrayList<>();
+  private static List<HospitalEdge> edgeList = new ArrayList<>();
```

Comment:
Do we want the implementation here to be an array list or something a little more efficient, like a linked list?